hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestLayoutOptimization.scala

```diff
@@ -111,8 +111,7 @@ class TestLayoutOptimization extends HoodieClientTestBase {
       .option("hoodie.clustering.inline.max.commits", "1")
       .option("hoodie.clustering.plan.strategy.target.file.max.bytes", "1073741824")
       .option("hoodie.clustering.plan.strategy.small.file.limit", "629145600")
-      .option("hoodie.clustering.plan.strategy.max.bytes.per.group", Long.MaxValue.toString)
```

Comment:
Let me explain the modification here: since we set the maximum size of the file written by clustering to `hoodie.clustering.plan.strategy.max.bytes.per.group`, if the value is Long.MaxValue.toString, after add by compressing ratio, it will be become negative, at this point the file will be split infinitely
![image](https://user-images.githubusercontent.com/37108074/211593944-7060014a-0792-4760-88cc-b6e46aa29f76.png)